k2/csrc/cuda/CMakeLists.txt

```diff
@@ -1,28 +1,15 @@
-add_library(context context.cu)
-target_include_directories(context PUBLIC ${CMAKE_SOURCE_DIR})
-target_link_libraries(context PUBLIC cub)
-target_link_libraries(context PUBLIC glog)
+include_directories(${CMAKE_SOURCE_DIR})
```

Comment:
No need to add this statement. It can be removed.

Reply:
Done.